aws.env.example

```diff
@@ -0,0 +1,11 @@
+AWS_ACCESS_KEY_ID=AKABCDEFGHIJKLMNOPQ
+AWS_SECRET_ACCESS_KEY=aBcDeFgHiJkLmNoPqRsTuVwXyZAbCdEfGhIjKlMn
+AWS_DEFAULT_REGION=us-east-1
+AWS_REGION=us-east-1
+
+CS_ORGANIZATION=ourorg
```

Comment:
Does this work? I thought the API call to Codeship required the organization's UUID (which comes back as part of the auth API call, if you need it).

Reply:
It has triggered the build for me several times.  It's also matches the codeship-go readme example.